admin_manual/conf.py

```diff
@@ -48,9 +48,9 @@
 # built documents.
 #
 # The short X.Y version.
-version = '10.0'
+version = '10.3'
```

Comment:
@PVince81 @settermjd are these really meant to be bumped to 10.3 ???

Reply:
Yep, I saw my mistake in the backport PR. Fixing now.